app/scripts/controllers/create/createFromImage.js

```diff
@@ -77,6 +77,8 @@ angular.module("openshiftConsole")
             portOptions: []
           };
           scope.labels = [];
+          scope.appLabel = {name: 'app', value: ''};
```

Comment:
nit: could be a local var rather than a scope var since it's not used in the view